discounts.go

```diff
@@ -0,0 +1,111 @@
+package httpshopify
+
+import (
+	"github.com/MOHC-LTD/shopify"
+)
+
```

Comment:
Petty comment. Can you add some comments above these functions/methods to keep it consistent with the other files. Apart from that, all looks good.